style.css

```diff
@@ -0,0 +1,201 @@
+  .wrap {
+    max-width: 1920px;
+    min-width: 375px;
+    height: 100vh;
+    margin: 0 auto;
+    background: url(./img/backround.jpg) no-repeat;
+    background-size: cover;
+    font-family: Museo;
+  }
+
+  .header {
+    width: 85%;
+    height: 200px;
+    margin: 0 auto;
+    display: flex;
+    align-items: center;
+  }
+
+  .header__logo {
+    justify-content: center;
+  }
+
+  .space {
+    flex: 2;
+  }
+
+  .menu {
+    flex: 2;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .menu__link {
+    color: #f8eee4;
+    text-decoration: none;
+    font-weight: bold;
+  }
+
+  .menu__link:hover {
+    color: grey;
+  }
+
+  @media all and (max-width: 600px) {
+    .menu__link {
+      visibility: hidden;
+    }
+  }
+
+  .burger {
+    width: 25px;
+    height: 14px;
+    position: relative;
+  }
+
+  .burger .bar {
+    padding: 0;
+    width: 25px;
+    height: 3px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/VeronikasBaliProject/pullRequest?prid=5927042)